integration_test/third_party_apps_test.go

```diff
@@ -460,7 +461,14 @@ func runLoggingTestCases(ctx context.Context, logger *logging.DirectoryLogger, v
 	return err
 }
 
-func runMetricsTestCases(ctx context.Context, logger *logging.DirectoryLogger, vm *gce.VM, metrics []*metadata.ExpectedMetric) error {
+type feature struct {
+	Module  string
+	Feature string
+	Key     string
+	Value   string
+}
+
+func runMetricsTestCases(ctx context.Context, logger *logging.DirectoryLogger, vm *gce.VM, metrics []*metadata.ExpectedMetric, fc feature_tracking.FeatureTrackingContainer, skipFeatureTracking bool) error {
```

Comment:
Do we really need two new parameters here? Why not just make `fc` nillable and use its nil-ness as a flag to skip?